aa/packages/smooth_app/ios/Runner/aa.lproj/InfoPlist.strings

```diff
@@ -0,0 +1 @@
+NSCameraUsageDescription = "This app needs camera access to scan barcodes and to take product photos";
```

Comment:
@g123k @monsieurtanuki @M123-dev  is this file used ? My new setup of crowdin has synced it for some reason. Do you notice anything weird in the English files of this PR ? Strings you have removed, or strings you have added but were never synced ?

Reply:
As far as I remember this file is used for the permission messages on iOS only.
I think one file is for the default message, and I can't remember if there is one file for all translations or one file per language.
Don't know if it was helpful.

Reply:
Crowdin seems to be creating duplicates of the file
![image](https://github.com/openfoodfacts/smooth-app/assets/1689815/a3e8897a-c59f-4fb7-8c5d-35e96a78534b)

Reply:
The file should go in packages/smooth_app/ios/Runner